Source/Widgets/Cesium3DTilesInspector/Cesium3DTilesInspector.js

```diff
@@ -221,27 +225,19 @@ define([
         return destroyObject(this);
     };
 
-    function makeSection(name, visibleProp, toggleProp, contents) {
-        var toggle = document.createElement('span');
-        toggle.className = 'cesium-cesiumInspector-toggleSwitch';
-        toggle.setAttribute('data-bind', 'text: ' + visibleProp + ' ? "-" : "+", click: ' + toggleProp);
-
+    function makeSection(panel, name, visibleProp, toggleProp, contents) {
```

Comment:
Synchronize implementation with `CesiumInspector::addSection`. Maybe a base class for both inspectors would be a good idea?